modules/extensions-spring-support/src/test/resources/models/heisenberg.json

```diff
@@ -32410,7 +32410,7 @@
           "namespace": "MULE"
         }
       },
-      "minMuleVersion": "4.5.0",
+      "minMuleVersion": "4.6.0",
```

Comment:
while doing the refactor I fixed a bug in the min mule version calculation for methods with parameter groups, where annotations inside parameter groups were not taken into account. I've checked models changes are they are correct. I'm copying here the reason for their new version:
`Method getInlinePersonalInfo has min mule version 4.6.0 because of its parameter info. Parameter info has min mule version 4.6.0 because it is of type PersonalInfo. Type PersonalInfo has min mule version 4.6.0 because of its field description. Field description has min mule version 4.6.0 because it is annotated with Text. Text was introduced in Mule 4.6.0.`